settlement/base.go

```diff
@@ -93,27 +93,11 @@ func (b *BaseLayerClient) Stop() error {
 	return nil
 }
 
-// SubmitBatch submits the batch to the settlement layer. This should create a transaction which (potentially)
-// triggers a state transition in the settlement layer.
-func (b *BaseLayerClient) SubmitBatch(batch *types.Batch, daClient da.Client, daResult *da.ResultSubmitBatch) *ResultSubmitBatch {
+// SubmitBatch tries submitting the batch in an async way to the settlement layer. Events are emitted on success or failure.
```

Comment:
How is that `async`?
You still call `d.client.BroadcastTx` and waits to see the tx result

Reply:
it's async by the [cosmos terms](https://tutorials.cosmos.network/academy/2-cosmos-concepts/3-transactions.html). submitting a tx in a broadcast mode async in cosmos means submitting the tx and not waiting to see if it was included in a block (but do get a response back). I can make it more clear that it's broadcast mode = async if you think it makes more sense.